src/test/java/io/debezium/connector/yugabytedb/TestHelper.java

```diff
@@ -380,9 +383,24 @@ public static Configuration.Builder getConfigBuilder(String namespaceName, Strin
                 .with(YugabyteDBConnectorConfig.STREAM_ID, dbStreamId);
     }
 
-    public static void setContainerHostPort(String host, int port) {
+    public static Configuration.Builder getConfigBuilderForCQL(String keyspaceName, String fullTableNameWithSchema, String dbStreamId) throws Exception {
+        return TestHelper.defaultConfig()
+                .with(YugabyteDBConnectorConfig.DATABASE_NAME, keyspaceName)
+                .with(YugabyteDBConnectorConfig.HOSTNAME,CONTAINER_YCQL_HOST)
+                .with(YugabyteDBConnectorConfig.PORT, CONTAINER_YCQL_PORT)
+                .with(YugabyteDBConnectorConfig.USER, "cassandra")
```

Comment:
We can add a static final variable named `DEFAULT_CASSANDRA_USER` with value `cassandra`